components/SubmissionError/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Text,
+  View
+} from 'react-native';
+import { Snackbar } from 'react-native-paper';
+
+import I18n from '../../modules/i18n';
+
+const SubmissionError = ({ error, setError }) => {
```

Comment:
@jamccomb92 something that would be great is if you passed the Snackbar's `text` as a props so that we can use this component for other errors

in fact (if you wanted) you could even rename the component `<ErrorSnack />` or `<PopUpError />` or something more general